aldryn_config.py

```diff
@@ -4,8 +4,8 @@
 
 class Form(forms.BaseForm):
 
-    show_all_recipients = forms.CheckboxField('Show all users in form recipients select', required=False)
+    show_all_recipients = forms.CheckboxField('Include only admin users as selectable e-mail recipients instead of everyone.', required=False)
 
     def to_settings(self, data, settings):
-        settings['ALDRYN_FORMS_SHOW_ALL_RECIPIENTS'] = data['show_all_recipients']
+        settings['ALDRYN_FORMS_SHOW_ALL_RECIPIENTS'] = not data['show_all_recipients']
```

Comment:
@FinalAngel what was the reason for negating the meaning? I don't see any logic change to make the change in text actually correct.

How about we call it ``Show all users as selectable e-mail in recipients (not only admins)``?